components/infobox/wikis/starcraft2/infobox_person_player_custom.lua

```diff
@@ -324,34 +314,48 @@ function CustomPlayer._getEarningsMedalsData(player)
 		playerConditions,
 		ConditionNode(ColumnName('date'), Comparator.neq, '1970-01-01 00:00:00'),
 		ConditionNode(ColumnName('liquipediatiertype'), Comparator.neq, 'Charity'),
-		ConditionNode(ColumnName('liquipediatiertype'), Comparator.neq, 'Qualifier'),
 		ConditionTree(BooleanOperator.any):add({
 			ConditionNode(ColumnName('individualprizemoney'), Comparator.gt, '0'),
+			ConditionNode(ColumnName('extradata_award'), Comparator.neq, ''),
 			ConditionTree(BooleanOperator.all):add({
 				ConditionNode(ColumnName('players_type'), Comparator.gt, 'solo'),
 				placementConditions,
 			}),
 		}),
 	})
 
-	local data = CustomPlayer._getLPDBrecursive(conditions:toString(), query, 'placement')
-
 	local earnings = {}
 	local medals = {}
 	earnings['total'] = {}
 	medals['total'] = {}
 	local earnings_total = 0
 
-	if type(data[1]) == 'table' then
-		for _, item in pairs(data) do
-			--handle earnings
-			earnings, earnings_total = CustomPlayer._addPlacementToEarnings(earnings, earnings_total, item)
+	local queryParameters = {
+		conditions = conditions:toString(),
+		order = 'liquipediatier asc, weight desc',
```

Comment:
Doesn't this need to be sorted on placements as second order? 
Say that `_MAXIMUM_NUMBER_OF_ACHIEVEMENTS` is `1`, and a player has two tournament finishes. One is 1st place in a Tier 2 tournament, and the other is a 4th place in a Tier 2 tournament. Due to higher prizepool, the 4th place tournament has a higher `weight`. If I'm thinking correctly, the 4th place tournament will be included, while it shouldn't be.
For an even more edge case, what is they have the same weight? It would be random if the 4th place tournament would be included.

Reply:
yeah, good catch, thx